<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumSettingsWidget</class>
 <widget class="QWidget" name="SpectrumSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>213</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>213</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frameVolume">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="3" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="volumeGreenSlider">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="volumeTotalSlider">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="4" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="volumeBlueSlider">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="volumeRedSlider">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>G</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Master</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Volume&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frameFrequencySpectrum">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="5">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QSpinBox" name="freqGminSpin">
        <property name="maximum">
         <number>40000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QSlider" name="freqBminSlider">
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>max</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QSpinBox" name="freqGmaxSpin">
        <property name="maximum">
         <number>40000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSlider" name="freqGminSlider">
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QSlider" name="freqBmaxSlider">
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>min</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="freqRminSpin">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>40000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QSpinBox" name="freqRmaxSpin">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>40000</number>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QSpinBox" name="freqBmaxSpin">
        <property name="maximum">
         <number>40000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSlider" name="freqGmaxSlider">
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSlider" name="freqRminSlider">
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QSpinBox" name="freqBminSpin">
        <property name="maximum">
         <number>40000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSlider" name="freqRmaxSlider">
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>max</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" rowspan="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>G</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>max</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>min</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>min</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Frequency Spectrum&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
